plugins/range/plugin.go

```diff
@@ -133,41 +132,52 @@ func setupRange4(args ...string) (handler.Handler4, error) {
 func setupRange(v6 bool, args ...string) (handler.Handler6, handler.Handler4, error) {
 	var err error
 	if len(args) < 4 {
-		return nil, nil, errors.New("need a file name, start of the IP range, end og the IP range and a lease time")
+		return nil, nil, fmt.Errorf("invalid number of arguments, want: 4 (file name, start IP, end IP, lease time), got: %d", len(args))
 	}
 	filename = args[0]
 	if filename == "" {
-		return nil, nil, errors.New("got empty file name")
+		return nil, nil, errors.New("file name cannot be empty")
 	}
 	ipRangeStart = net.ParseIP(args[1])
 	if ipRangeStart.To4() == nil {
-		return nil, nil, errors.New("expected an IP address, got: " + args[1])
+		return nil, nil, fmt.Errorf("invalid IPv4 address: %v", args[1])
 	}
 	ipRangeEnd = net.ParseIP(args[2])
 	if ipRangeEnd.To4() == nil {
-		return nil, nil, errors.New("expected an IP address, got: " + args[2])
+		return nil, nil, fmt.Errorf("invalid IPv4 address: %v", args[2])
 	}
 	if binary.BigEndian.Uint32(ipRangeStart.To4()) >= binary.BigEndian.Uint32(ipRangeEnd.To4()) {
-		return nil, nil, errors.New("start of IP range has to be lower than the end fo an IP range")
+		return nil, nil, errors.New("start of IP range has to be lower than the end of an IP range")
 	}
 	LeaseTime, err = time.ParseDuration(args[3])
 	if err != nil {
-		return Handler6, Handler4, errors.New("expected an uint32, got: " + args[3])
+		return Handler6, Handler4, fmt.Errorf("invalid duration: %v", args[3])
+	}
+	r, err := os.Open(filename)
```

Comment:
This file is not being closed and I think it is not being used either.

Reply:
related to the above